<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gsf</class>
 <widget class="QDialog" name="gsf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="temprature">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>1061</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Gestion stock</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>971</width>
       <height>651</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;taief.jpg&quot;);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>date_stk</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>290</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantite">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>220</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>290</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>340</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="dat">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>256</width>
         <height>190</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tab_stk_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>430</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>date_stk</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>370</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>510</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>quatite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>370</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_Stock</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>470</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>480</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantite_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>510</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>190</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ToolTipBase">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ToolTipBase">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ToolTipBase">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff1c3e;&quot;&gt;TRIE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>quantite</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>statistique</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="dat_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>379</y>
         <width>301</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_2">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>500</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refresh">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>530</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rechercher">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>560</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dat_3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>IMPRIMER</string>
       </property>
      </widget>
      <widget class="QComboBox" name="recherche">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>20</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>quantite</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="rechercher_3">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>10</y>
         <width>172</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>470</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>IMPORT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="EXPORT">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>440</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>EXPORT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="excel">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>410</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ACCESS_FILE</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestion  fournisseurs</string>
    </attribute>
    <widget class="QTabWidget" name="affichage2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>601</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter1">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="adresse">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tab_fourn_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>441</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adresse</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="delete_2">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stat_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>statistique</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_f</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>330</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>360</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>390</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>360</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>390</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>370</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>430</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="adresse_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>30</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TRIE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refresh_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>410</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rechercher_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>460</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="com">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adresse</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="rechercher_4">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton_2">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>60</y>
         <width>172</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>IMPRIMER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="IMPORT">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>330</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>IMPORT</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
